--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,15 +16,9 @@
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake/")
 
-# set version based on git
-find_package(Git)
-if(GIT_FOUND)
-    EXECUTE_PROCESS(COMMAND ${GIT_EXECUTABLE} describe --tags --always --dirty OUTPUT_VARIABLE VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
-    STRING(REGEX REPLACE "^v(.*)" "\\1" VERSION "${VERSION}")
-    add_definitions("-DVERSION=\"${VERSION}\"")
-endif()
+# set version based on environment variable
+add_definitions("-DVERSION=\"${APK_VERSION}\"")
 
-
 # define executable to build
 include_directories("inc")
 add_executable(${PROJECT_NAME} main.cpp)
@@ -112,7 +106,6 @@
 endif()
 
 # v4l2wrapper
-EXEC_PROGRAM("git submodule update --init")
 add_subdirectory(v4l2wrapper)
 include_directories("v4l2wrapper/inc")
 target_link_libraries (${PROJECT_NAME} v4l2wrapper)
